src/goalServices/getGoalsMissingDataForActivityReportSubmission.test.js

```diff
@@ -46,21 +46,21 @@ describe('getGoalsMissingDataForActivityReportSubmission', () => {
       status: GOAL_STATUS.IN_PROGRESS,
       name: goalTitle,
       grantId: activeGrant.id,
-      goalTemplateId: template.id,
+      // goalTemplateId: template.id,
```

Comment:
I feel like there should be some explanation of why we're commenting this out. e.g. "having template ID set will cause index collisions unless we split out the grants, which needlessly complicates these tests" or something like that

Reply:
You already took care of this!